css/components/button.css

```diff
@@ -0,0 +1,112 @@
+.btn {
+    cursor: pointer;
+    font-size: 1.6rem;
+    padding: 0.75em 1em;
+    margin: 0.2em;
```

Comment:
one suggestion try to use the same CSS units everywhere.

Reply:
In this particular case, I wanted the padding and margin to be proportional to the text size of the button whose font size is independent of its parent. So, I feel using `em` was appropriate here.